gamestonk_terminal/cryptocurrency/defi/defi_controller.py

```diff
@@ -165,6 +189,275 @@ def call_ayr(self, other_args: List[str]):
+            help="Number of proposals to show",
+            default=10,
+        )
+        parser.add_argument(
+            "-s",
+            "--sort",
+            dest="sortby",
+            type=str,
+            help="Sort by given column. Default: id",
+            default="id",
+            choices=[
+                "submitTime",
+                "id",
+                "depositEndTime",
+                "status",
+                "type",
+                "title",
+                "Yes",
+                "No",
+            ],
+        )
+        parser.add_argument(
+            "--status",
+            dest="status",
+            type=str,
+            help="Status of proposal. Default: voting",
+            default="voting",
+            choices=["voting", "deposit", "passed", "rejected"],
+        )
+        parser.add_argument(
+            "--descend",
+            action="store_false",
+            help="Flag to sort in descending order (lowest first)",
+            dest="descend",
+            default=False,
+        )
+
+        ns_parser = parse_known_args_and_warn(
+            parser, other_args, EXPORT_ONLY_RAW_DATA_ALLOWED
+        )
+
+        if ns_parser:
+            terramoney_fcd_view.display_gov_proposals(
+                status=ns_parser.status,
+                export=ns_parser.export,
+                sortby=ns_parser.sortby,
+                descend=ns_parser.descend,
+                top=ns_parser.limit,
+            )
+
+    def call_gacc(self, other_args: List[str]):
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="gacc",
+            description="""
+                Displays terra blockchain account growth history.
+                [Source: https://fcd.terra.dev/swagger]
+            """,
+        )
+        parser.add_argument(
+            "-l",
+            "--limit",
+            dest="limit",
+            type=check_positive,
+            help="Number of days to show",
+            default=90,
+        )
+        parser.add_argument(
+            "--cumulative",
+            action="store_false",
+            help="Show cumulative or discrete values. For active accounts only discrete value are available",
+            dest="cumulative",
+            default=True,
+        )
+        parser.add_argument(
+            "-k",
+            "--kind",
+            dest="kind",
+            type=str,
+            help="Total account count or active account count. Default: total",
+            default="total",
+            choices=["active", "total"],
+        )
+        parser.add_argument(
+            "--descend",
+            action="store_false",
+            help="Flag to sort in descending order (lowest first)",
+            dest="descend",
+            default=False,
+        )
+
+        ns_parser = parse_known_args_and_warn(
+            parser, other_args, EXPORT_ONLY_RAW_DATA_ALLOWED
```

Comment:
replace `EXPORT_ONLY_RAW_DATA_ALLOWED` with `EXPORT_BOTH_RAW_DATA_AND_FIGURES` since chart is returned